### PR TITLE
basic entities creation: Inventory, Item, Menu, Order, Product

### PR DESCRIPTION
**Pull request by gmzjohn**

### CODE REVIEW

File: src/entities/Inventory.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Item } from './Item'
+
+export class Inventory {
+    id: Number;
+    item: [Item];
+}
```
Comment (reynoldmorel) [inline src/entities/Inventory.tsx:6]: Please add an enter to avoid the warning. Do for all classes

File: src/entities/Inventory.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Item } from './Item'
+
+export class Inventory {
```
Comment (reynoldmorel) [inline src/entities/Inventory.tsx:3]: Always use `export default` if there will be only one class, function or type exported

**Review by reynoldmorel — COMMENTED** [review]
Please add the UML diagram.

Also please separate your commit in different PRs, I see `order`, `item`, `product`, `menu` in the same PR.

It's difficult to give you a review of all of them in just one PR.

**Comment by gmzjohn** [comment]
Thanks for the feedback, I added the changes to the PR.

**Pull request merged**